daemon/graphdriver/devmapper/driver.go

```diff
@@ -160,7 +161,21 @@ func (d *Driver) Remove(id string) error {
 	if err := d.DeviceSet.DeleteDevice(id, false); err != nil {
 		return fmt.Errorf("failed to remove device %s: %v", id, err)
 	}
-	return system.EnsureRemoveAll(path.Join(d.home, "mnt", id))
+
+	// Most probably the mount point is already removed on Put()
+	// (see DeviceSet.UnmountDevice()), but just in case it was not
+	// let's try to remove it here as well, ignoring errors as with
+	// an older kernel can return EBUSY if e.g. the mount was leaked
+	// to other mount namespaces. A failure to remove the container's
+	// mount point is not important and should not be treated
+	// as a failure to remove the container.
+	mp := path.Join(d.home, "mnt", id)
+	err := unix.Rmdir(mp)
+	if err != nil && !os.IsNotExist(err) {
+		logrus.Warnf("Can't rmdir %s: %s", mp, err)
```

Comment:
perhaps

```go
logrus.Warnf("unable to remove mountpoint %q: %s", mp, err)
```

("unable" or "failed" - used "unable" to make it look less as an important error)

Reply:
`WithField("module", "graphdriver/devicemapper")`